### ISSUE TITLE
Is the mod compatible with pollution of the realms just like vanilla create?

### ISSUE THREAD

**Issue by NeroWT** [george8188625/Create-Diesel-Generators#96]
If not it'd be a cool thing to add since combustion engines definitely are the most polutant things in the mod

**Comment by george8188625** [comment]
I don't think it is. unfortunately, as far as I'm aware potr doesn't provide a public api, they do compat themselves. so there is nothing I can do.
